### PR TITLE
/getting-started/setup-availability the table of days breaks depending on the language

### LINKED ISSUE #4993: /getting-started/setup-availability the table of days breaks depending on the language
Found a bug? Please fill out the sections below. 👍

### Issue Summary

When we are configuring our account for the first time when opening the site, depending on the language that the user uses, the days table can break.

English Version: 
Captura de tela de 2022-10-13 15-59-16

Portuguese Version:
Captura de tela de 2022-10-13 15-38-54

### PR DESCRIPTION
**Pull request by gitstart**
## What does this PR do?

Fixes #4993

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Demo
Old Version: 
Captura de tela de 2022-10-13 15-38-54

New Version:
Captura de tela de 2022-10-13 16-19-41

### CODE REVIEW

File: packages/features/schedules/components/Schedule.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ const ScheduleDay = <TFieldValues extends FieldValues>({
   return (
     <div className="mb-1 flex w-full flex-col py-1 sm:flex-row">
       {/* Label & switch container */}
-      <div className="flex h-11 items-center justify-between">
+      <div className="flex h-11 w-32 items-center justify-between break-normal">
```
Comment (JeroenReumkens) [inline packages/features/schedules/components/Schedule.tsx:49]: @gitstart `break-normal` is the default if I'm correct, right? Does add anything here? 
Cool change apart from that, thanks a lot! 🙏

Comment (JeroenReumkens) [reply packages/features/schedules/components/Schedule.tsx:49]: Could you also make this full width on mobile, and only `w-32` starting at a bigger breakpoint? Makes it a bit nicer on mobile I think!

**Review by JeroenReumkens — DISMISSED** [review]
Thank you so much for you work @gitstart  Really valuable change. I added 2 small requests for you that would make it even better I think 🙏

**Comment by gitstart** [comment]
Thank you for the review @JeroenReumkens. Requested changes has been made, please take a look

**Demo**
https://www.loom.com/share/673092b6806f444d96656ac40a630b98

**Review by PeerRich — APPROVED** [review]

**Pull request merged**